PocketKit/Sources/PocketKit/Collections/CollectionViewModel.swift

```diff
@@ -11,6 +11,31 @@ import Localization
 import Analytics
 import Network
 
+enum ReadableType {
```

Comment:
This code used to reside in `HomeViewModel` and moved here since it's now the only place where it's used.